tests/common/include/tests/common/tools/simulator.hpp

```diff
@@ -61,7 +61,7 @@ struct simulator {
                     interactor);
                 typename parameter_resetter<transform3>::state resetter{};
                 typename actor_chain_type::state actor_states = std::tie(
-                    transporter, interactor, scatterer, writer, resetter);
+                    transporter, interactor, scatterer, resetter, writer);
```

Comment:
Just as a comment: The ordering here does not matter, but of course it is more readable this way :)

Reply:
Wait - ordering doesn't matter in actor chain?

Reply:
How do I order the actor execution then?

Reply:
Just not in the actor state tuple, since the states are fetched by type, if I remember correctly.

Reply:
Sorry, should have been more precise and it is also really just as an aside

Reply:
In the actor chain itself, the ordering does matter of course!

Reply:
I see. Think I understand

Reply:
Yes, sorry, I should not have made that comment, it was really not substantial :D